app/scripts/views/force_auth.js

```diff
@@ -43,7 +43,7 @@ define(function (require, exports, module) {
       return {
         email: email,
         fatalError: getFatalErrorMessage(this, fatalError),
-        isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
+       // isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
```

Comment:
Similar here, this line should be removed from the source code, with the comma on the previous line removed.